plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -352,9 +355,12 @@ const config: ControlPanelConfig = {
             config: {
               type: 'CheckboxControl',
               label: t('Show Max Label'),
-              default: xAxisShowMaxLabel,
+              default: xAxisShowMinLabel,
```

Comment:
Shouldn't this be unchanged?

Reply:
em, I thought `default ` should be removed. I will correct it.